Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -213,6 +214,7 @@ namespace Multiplayer
     {
         required.push_back(AZ_CRC_CE("NetworkingService"));
         required.push_back(AZ_CRC_CE("MultiplayerStatSystemComponent"));
+        required.push_back(AZ_CRC_CE("PhysicsService"));
```

Comment:
To me this it seems odd to couple Physx + Multiplayer together just to get stats.

Did we consider just adding metric to PhysX metrics? Feels there should be other ways to measure this having to directly observe PhysX.

Reply:
We already depend on the physics gem for its components. This is specifically to make sure when the system component calls for:
```
        if (auto* physXSystem = PhysX::GetPhysXSystem())
        {
            physXSystem->RegisterPreSimulateEvent(m_preSimulateHandler);
            physXSystem->RegisterPostSimulateEvent(m_postSimulateHandler);
        }
```
That the system is available.

Reply:
I moved the registration to a later time, and now this explicit dependency has been removed.